src/lib/pages/validator-details/components/validator-overview/VotingPowerOverview.tsx

```diff
@@ -56,47 +172,56 @@ export const VotingPowerOverview = () => {
             Voting Power
           </Heading>
         </Flex>
-        <ValueWithIcon icon="vote" value="7.21%" />
+        <ValueWithIcon icon="vote" value={votingPowerPercent} />
         <Flex gap={2} alignItems="center" mt={1}>
           <Flex direction="column">
             <Text fontWeight={700} variant="body1">
-              123,456,789.0{" "}
+              {votingPowerAmount}{" "}
               <span
                 style={{
                   fontWeight: "400",
                 }}
               >
-                OSMO
+                {assetInfo?.symbol ?? ""}
```

Comment:
Ask to designer what should we display if symbol is undefined.

Reply:
`{getTokenLabel(singleStakingDenom, assetInfo?.symbol)}`